arch/arm/src/nrf52/hardware/nrf52_clock.h

```diff
@@ -60,7 +75,7 @@
 #define NRF52_CLOCK_EVENTS_LFCLKSTARTED_OFFSET 0x0104 /* LFCLK started */
 #define NRF52_CLOCK_EVENTS_DONE_OFFSET         0x010c /* Calibration of LFCLK RC oscillator complete event */
 #define NRF52_CLOCK_EVENTS_CTTO_OFFSET         0x0110 /* Calibration timer timeout */
-#ifdef CONFIG_ARCH_CHIP_NRF52840
+#ifdef HAVE_CLOCK_CALTIMER
```

Comment:
Maybe rename this to HAVE_CLOCK_CALTIMER_EVENTS since nrf52832 has a calibration timer but only does not generate these events.

Reply:
DONE